.ado/xcconfig/publish_overrides.xcconfig

```diff
@@ -1,2 +1,13 @@
+// For publish builds, only provide line tables for symbolizing crashes
+CLANG_DEBUG_INFORMATION_LEVEL=line-tables-only
+// OTHER_SWIFT_FLAGS=-gline-tables-only
```

Comment:
@markavitale do you know what the impact of not including this build flag may be? I copied these flags from FluentUI Apple, and I suppose we were never building with it to begin with so it should be alright. I was confused why this caused build failures earlier though.

Reply:
Does the xcconfig on line 2 just do a separate but very similar thing?
`CLANG_DEBUG_INFORMATION_LEVEL=line-tables-only`

It also sounds like it's a limiter so we are just getting more info than we need?

Reply:
Yep, one is for clang compiler (Objective-C, C, C++, etc), and the other is for the Swift compiler (Swift code). We probably can't ship something with full debug information (which includes things like filepaths of the machines that did the compilation), so the alternative would be dropping the ability to symbolize crashes involving FURN.

Reply:
Just gonna drop this here for reference. https://gist.github.com/CodaFi/b1e0d0f37cd2890c07c5

Reply:
To be overcautious, I added `-gline-tables-only` to every pod spec to be over-cautious. Looking at a local build log, it also seemed that the greater project `ReactTestApp` was also inheriting that arg from elsewhere. Between those two, I feel this is safer now. I could probably revisit how to inherit xcconfig args more cleanly here, but 🤷🏾